trollmoves/dispatcher.py

```diff
@@ -241,30 +247,41 @@ class DispatchConfig(YAMLConfig):
     """Class to handle dispatch configs."""
 
     def __init__(self, filename, callback):
+        """Initialize dispatch configuration class."""
         self.callback = callback
         super().__init__(filename)
 
     def read_config(self):
+        """Read configuration file."""
         super().read_config()
         self.callback(self.config)
 
 
 class Dispatcher(Thread):
     """Class that dispatches files."""
 
-    def __init__(self, config_file):
+    def __init__(self, config_file, publish_port=None,
+                 publish_nameservers=None):
```

Comment:
Could you test that not providing the nameservers on the command line actually gives None, and in that case the regular multicast system is used?

Reply:
Done.